scripts/portstat

```diff
@@ -204,6 +205,76 @@ def parse_bcmcmd_ps(bcmcmd_output):
     bcmcmd_lines = bcmcmd_output.split("\n")
 
 
+def get_cnstat(cnstat_dir):
```

Comment:
it does not look the cnstat_dir is really used in this function, can you remove this argument?